bugsnag-android-core/src/main/java/com/bugsnag/android/ImmutableConfig.kt

```diff
@@ -32,7 +32,7 @@ internal data class ImmutableConfig(
 
     companion object {
         private const val HEADER_API_PAYLOAD_VERSION = "Bugsnag-Payload-Version"
-        private const val HEADER_API_KEY = "Bugsnag-Api-Key"
+        internal const val HEADER_API_KEY = "Bugsnag-Api-Key"
```

Comment:
I'm not sure why this change is required.

Reply:
`HEADER_API_KEY` is now accessed from the `InternalReportDelegate` rather than using a string constant. Because this is in another class it needs to be `internal` rather than `private` to be accessed.

Reply:
The usage of this constant within `InternalReportDelegate` seemed to have escaped the changeset - I've updated this to replace the string constant with `HEADER_API_KEY`